src/services/models/authModels.js

```diff
@@ -21,7 +21,10 @@ export const permissions = {
 
   createJobs: 'CreateJobs',
 
-  updateSIMManagement: 'UpdateSIMManagement'
+  updateSIMManagement: 'UpdateSIMManagement',
+
+  deletePackages: 'DeletePackages',
+  addPackage: 'AddPackage'
```

Comment:
All of the other permission names are plural. This one should be too.

Reply:
changed